ecc/bn254/fr/poseidon/poseidon.go

```diff
@@ -0,0 +1,111 @@
+package poseidon
+
+import (
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr/poseidon/constants"
+
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
+)
+
+func zeroElement() *fr.Element {
```

Comment:
There seems no difference between `fr.Zero()` and `zeroElement()`. we can use `fr.Zero()` directly

Reply:
`fr.Zero()` was added by me to `element` package but since it won't be used elsewhere. I decided to write it in `poseidon` package

Reply:
ok, fine for me